drivers/flash/soc_flash_nrf.c

```diff
@@ -23,15 +23,30 @@
 #include "controller/ticker/ticker.h"
 #include "controller/include/ll.h"
 
+#if defined(CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE)
+
+/* The timeout is multiplied by 1.5 because switching tasks may take
+ * significant portion of time.
+ */
+#define FLASH_TIMEOUT_MS ((FLASH_PAGE_ERASE_MAX_TIME_US)\
+			* (FLASH_PAGE_MAX_CNT) / 1000 * 15 / 10)
+
+#else
+
+#define FLASH_TIMEOUT_MS ((FLASH_PAGE_ERASE_MAX_TIME_US)\
+			* (FLASH_PAGE_MAX_CNT) / 1000)
+
+#endif	/* CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE */
```

Comment:
It would probably look a bit better with this tab character before `/*` replaced with one space.

Reply:
OK